tests/app/phase-tester-response/issue-pending/issue-pending.component.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { IssuePendingComponent } from '../../../../src/app/phase-tester-response/issue-pending/issue-pending.component';
+import { Issue, STATUS } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { Team } from '../../../../src/app/core/models/team.model';
+import { IssueComment } from '../../../../src/app/core/models/comment.model';
+import { EMPTY_TEAM_RESPONSE, PENDING_TUTOR_MODERATION } from '../../../constants/githubcomment.constants';
+import { TutorModerationTodoTemplate } from '../../../../src/app/core/models/templates/tutor-moderation-todo-template.model';
+
+describe('IssuePendingComponent', () => {
+    describe('.ngOnInit()', () => {
+        const dummyTeam: Team = new Team({
+            id: 'dummyId',
+            teamMembers: [],
+          });
+        let dummyIssue: Issue;
+        let issuePendingComponent: IssuePendingComponent;
+        const issueComment: IssueComment = new TutorModerationTodoTemplate([PENDING_TUTOR_MODERATION]).comment;
+
+        beforeEach(() => {
+            dummyIssue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+            issuePendingComponent = new IssuePendingComponent();
+            issuePendingComponent.ngOnInit();
```

Comment:
Since the `filter` function is stateless, I think we can shift this to the outer scope so that it only needs to be run once for this entire suite :)